functionnal_tests/cypress/integration/activity_feed/common.spec.js

```diff
@@ -71,35 +71,35 @@ for (const pageTestCase of activityPages) {
           .should('contain.text', 'commented')
       })
 
+      /* FIXME - GB - 2020-12-29 - this test is unstable and it will be fixed at https://github.com/tracim/tracim/issues/3392
       it('should be reordered only when the "Refresh" button is pressed', () => {
```

Comment:
I suggest to use `it.skip` instead of commenting the test. This allows Cypress to show the number of disabled tests.

Reply:
changed